kubemarine/kubernetes/__init__.py

```diff
@@ -382,6 +382,8 @@ def join_control_plane(cluster: KubernetesCluster, node: NodeGroup, join_dict: d
 
     components.wait_for_pods(node)
 
+    # modifies file permissions for kubelet configuration file
+    node.sudo("chmod 600 /var/lib/kubelet/config.yaml")
```

Comment:
Although `/var/lib/kubelet/config.yaml` currently has 644 permissions, `/var/lib/kubelet` has 700. So the "effective" permissions for `/var/lib/kubelet/config.yaml` is 700 & 644 = 600. In particular, no user apart from root can read the file.
Can the above statement not always be true in our installation?